app/models/project.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Project < ApplicationRecord
+  validates :name, presence: true, uniqueness: true, length: { minimum: 1, maximum: 60 }
```

Comment:
Move 1 and 60 to constant. Also, [you can use range `1..60` here](https://guides.rubyonrails.org/active_record_validations.html#length).